src/main/java/com/porpapps/wealthwatchapi/.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You need to delete these files, they shouldn't be here inside the package

Reply:
done.